png_to_asm.py

```diff
@@ -23,6 +25,12 @@ def load_image(src: str) -> List[Color]:
 
 
     tiles = [extract_tile(image, x, y) for y in range(n_h) for x in range(n_w)]
+    global color_list
+    temp = []
+    for l in tiles:
+        temp += l
+    color_list = list(set(temp))
+    print(color_list)
```

Comment:
Either add a global flag for overriding `color_list` or check that `color_list` is empty beforehand